nibabel/minc2.py

```diff
@@ -77,7 +77,7 @@ def _get_dimensions(self, var):
             dimorder = var.attrs['dimorder'].decode()
         except KeyError:  # No specified dimensions
             return []
-        return dimorder.split(',')
+        return dimorder.split(',')[:len(var.shape)]
```

Comment:
Test case would be good.  Could you add a comment explaining as well?   Otherwise it might be puzzling for someone coming fresh to the code.